<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QMainWindow" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/gamenew.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="fireman">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>54</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/fireman.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="blue1">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>520</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="blue3">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="blue2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>285</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="blue4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="red1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="red2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>265</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="red3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="red4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="watergirl">
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/image/watergirl.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
